### ISSUE TITLE
Thoughts on science questions and experimental design

### ISSUE THREAD

**Issue by bpbond** [PNNL-PREMIS/SLA#5]
Hi @lilliehaddock - here are some thoughts on how I've been mentally framing your work. We can discuss in person, but hopefully putting these here will be useful during background reading and planning.

**Introduction/science question**

_First_, climate change is expected to produce rising sea levels and increased storm surges into coastal ecosystem.
[Many references]

_Second_, exposure to salinity is stressful for plants, and produces physiological and morphological changes in the laboratory.
https://www.nrcresearchpress.com/doi/abs/10.1139/b02-018#.XV2V1i2ZMxE
https://www.sciencedirect.com/science/article/pii/S0098847204000413
https://academic.oup.com/aob/article/119/1/1/2670018
http://www.cropj.com/carvalcante_12_4_2018_563_571.pdf
(Here and below, these are quick googling references for example; I don't guarantee most appropriate ones for your work.)

_Third_, specific leaf area (SLA) is a critical structural and physiological parameter...
[basically from your SULI report, fleshed out with references]

_Fourth_, but it's unclear how this will play out in real-world field conditions. This introduces substantial uncertainty into predicting how coastal forests will respond (in stress, mortality, and biogeochemical cycling) to rising sea levels and increased storm surges.
https://www.sciencedirect.com/science/article/pii/S0921818115301879
https://www.sciencedirect.com/science/article/pii/S0006320716303007
https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2017JG004125
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0132079
Also cite Nick Ward's 2019 PREMIS paper!

We conducted an observational study of SLA in coastal deciduous forests at the Smithsonian Environmental Research Center (SERC) in Maryland, USA, taking advantage of a natural gradient in salinity along a tidal creek. Our goals were to (i) characterize SLA across a broad range of species and canopy heights in these forests; (ii) examine canopy structural differences in near-shore and upland trees; and (iii) infer how changes in future salinity might affect SLA, and thus canopy physiology, by comparing same-species SLA values along the creek's salinity gradient.

**Comment by bpbond** [comment]
**Methods**

1. Describe SERC ecosystem - this can be lifted (with rewording into your own words of course!) from @stephpenn1 and my papers
2. Describe three forest stands along Muddy Creek, varying in salinity exposure, each with two sampling areas: creekside (high moisture and salinity exposure) and upland (low).
3. Sampling: when, how, number of species, instruments used, calculations.
4. Leaf litter: sampling, how we used litterfall (g/m2) times SLA (cm2/g) to compute leaf area index (LAI, m2/m2).
5. Statistics: how we tested for upland versus lowland differences; species differences; and differences along Muddy Creek. @bpbond writes this with you
6. Data availability (mention TRY!)
